universal-login-relayer/test/integration/core/services/MultiSignatureExecute.js

```diff
@@ -56,9 +56,12 @@ describe('INT: MultiSignatureExecute', async () => {
       const signedMessage0 = createSignedMessage(msg, wallet.privateKey);
       const signedMessage1 = createSignedMessage(msg, actionKey);
       await messageHandler.handleMessage(signedMessage0);
-      await messageHandler.handleMessage(signedMessage1);
+      const {messageHash} = await messageHandler.handleMessage(signedMessage1);
       await messageHandler.stopLater();
       expect(await provider.getBalance(msg.to)).to.eq(expectedBalance);
+      const {state, transactionHash} = await messageHandler.getStatus(messageHash);
+      expect(transactionHash).to.not.be.null;
```

Comment:
Please something more strict, at least:
`expect(transactionHash).to.be.properHex(...);`